src/EVM/SymExec.hs

```diff
@@ -911,10 +953,24 @@ subBufs model b = Map.foldlWithKey subBuf b model
                       else a
           e -> e
 
-subStore :: Map W256 (Map W256 W256) -> Expr a -> Expr a
-subStore model b = mapExpr go b
+subStores :: Map (Expr EAddr) (Map W256 W256) -> Expr a -> Expr a
```

Comment:
Maybe this function could have a bit of a comment about what it's doing. Its function signature does not indicate it to be something trivial :S